tests/scenarios.go

```diff
@@ -406,9 +409,14 @@ func getRequestBuilder(ctx gobdd.Context) (reflect.Value, error) {
 
 	// first argument is always context.Context
 	in[0] = reflect.ValueOf(GetCtx(ctx))
+	requestArgs := GetRequestArguments(ctx)
 	for i := 1; i < f.Type().NumIn(); i++ {
-		object := GetRequestArguments(ctx)[i-1]
-		in[i] = object.(reflect.Value)
+		if len(requestArgs) > i-1 {
+			object := requestArgs[i-1]
+			in[i] = object.(reflect.Value)
+		} else {
+			return f, nil
+		}
```

Comment:
Would make more sense to check before the loop if `len(requestArgs) >= f.Type().NumIn()`